include/glow/Graph/Graph.h

```diff
@@ -31,6 +31,7 @@
 
 namespace glow {
 class PlaceholderBindings;
+class SaveNode;
```

Comment:
I think this is not necessary. The full definition of this class is already included above thanks to `Nodes.h`.